Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "64e4ac1c23a21b1db62991b034ac7d38717874ef14675553d0d7cd27d01d6797"
```

Comment:
I may be completely misunderstanding this, but why are so many of the hashes and versions being changed in the Pipfile.lock? Is that because of the one change to the Pipfile?

Apologies if i'm missing something.

Reply:
There could be many reasons for that. Apart from the new dependency being added, already existing dependencies could have been updated by their developers which could result in different versions and hashes.

Reply:
Oh okay that makes sense thanks @noorbuchi